File: age_verifier.js
var age_verifier = function(age) {
  if (age < 21) {
    console.log("too young");
  } else {
    console.log("proceed");
  }
}

console.log("I am 18. May I come in?")
age_verifier(18);

console.log("Haha! :-p  I am 25. May I come in?")
age_verifier(25);

File: age_verifier.rb
def age_verifier (age)
  if age < 21
    puts "too young"
  else
    puts "proceed"
  end
end

puts "I am 18. May I come in?"
age_verifier(18)

puts "Haha! :-p  I am 25. May I come in?"
age_verifier(25)


# run it from command line
# ruby -r "./age_verifier" -e "age_verifier(12)"
